frontend/src/representations/RepresentationsWebSocketContainer.tsx

```diff
@@ -38,7 +37,7 @@ import {
   ShowToastEvent,
   SwitchSelectionEvent,
 } from './RepresentationsWebSocketContainerMachine';
-const representationsEventSubscription = gql`
+const representationsEventSubscription = gql(`
```

Comment:
are these parenthesis really useful?

Reply:
Ok so it's a bit subtle but they are indeed useful and required for the `FormWebSocketContainer`